tests/pages/services/ServiceActions-cy.js

```diff
@@ -59,7 +61,8 @@ describe('Service Actions', function () {
           nodeHealth: true
         });
 
-        cy.visitUrl({url: '/services/overview/%2Fsleep'});
+        cy.visitUrl({url: '/services/overview/%2Fsleep'})
+          .wait(['@marathon/groups', '@marathon/deployments', '@state']);
```

Comment:
I feel we should just rely on what's on the page to tell if it's ready or not rather than relying on endpoints responding. Can't we adjust the page to only render once we've received data from these different endpoints and showing the loading indicator until this data has been received?